src/features/reducer.js

```diff
@@ -0,0 +1,15 @@
+import { combineReducers } from 'redux';
+import { reducer as form } from 'redux-form';
+
+import { reducer as list } from './tool-search/tool-list';
+import { reducer as edit } from './tool-search/tool-edit';
+import client from '../core/apollo-client';
+
+const rootReducer = combineReducers({
+  form,
+  list,
+  edit,
+  apollo: client.reducer(),
+});
+
+export default rootReducer
```

Comment:
this is the `root-reducer`, I would rename it and move to `core` (you already have `root-reducer` there but it is empty)

Reply:
I moved root-reducer to core folder